benches/src/is_prime.rs

```diff
@@ -0,0 +1,101 @@
+//! Benchmark for the is_prime program
+//!
+//! This benchmark compiles the is_prime Rust program to Miden assembly
+//! and measures its execution performance in the Miden VM.
+
+use std::path::PathBuf;
+
+use anyhow::Result;
+use clap::{Arg, Command};
+use miden_compiler_benches::BenchmarkRunner;
+
+fn main() -> Result<()> {
+    let matches = Command::new("is_prime_benchmark")
```

Comment:
I'd recommend defining an argument structure, like so:

```rust
use clap::Parser;

#[derive(Parser)]
struct Config {
    /// The number to test for primality
    #[arg(short = 'i', long, value_name = "NUMBER", default_value = "29")]
    input: usize,
    /// Path to the is_prime source file
    #[arg(short = 's', long, value_name = "PATH", default_value = "examples/is_prime/src/lib.rs")]
    source: PathBuf,
    /// Number of iterations to run
    #[arg(short = 'n', long, value_name = "COUNT", default_value = "1")]
    iterations: usize,
}
```

Then you can parse the arguments with just:

```rust
fn main() -> anyhow::Result<()> {
    let config = Config::parse()?;
    ...
}
```